lib/core/src/rcMisc.cpp

```diff
@@ -4725,7 +4726,8 @@ int load_in_addr_from_hostname(const char* _hostname, struct in_addr* _out)
     *_out = reinterpret_cast<struct sockaddr_in*>(p_addrinfo->ai_addr)->sin_addr;
 
     if (CLIENT_PT != ::ProcessType) {
-        const std::chrono::seconds age{irods::get_dns_cache_eviction_age()};
+        const auto cache_config = irods::get_advanced_setting<nlohmann::json>(irods::KW_CFG_DNS_CACHE);
+        const std::chrono::seconds age{cache_config.at(irods::KW_CFG_EVICTION_AGE_IN_SECONDS).get<int>()};
```

Comment:
I realize string allocations can throw and we are leaning on the schema validation, but I think we should handle exceptions around this.

Reply:
Agreed. The original code handled exceptions and this should do the same.

Reply:
Done.